integreat_cms/firebase_api/firebase_api_client.py

```diff
@@ -41,7 +41,7 @@ def __init__(self, push_notification):
         self.prepared_pnts = []
         self.primary_pnt = PushNotificationTranslation.objects.get(
             push_notification=push_notification,
-            language=push_notification.region.default_language,
+            language=push_notification.regions.first().default_language,
```

Comment:
I actually don't quite get what the difference between primary and secondary PNs is here in the end. Is this only about using the title in the default language as a fallback? Why do we only check for the `title` in line 73 but also set `text` in line 77?

Reply:
> However, I'm also not sure why both fields are replaced if only one field is unset.

There is no reason. It is just convention: if the title is not set, the notification is sent in the primary/source/main language.

Reply:
Basically one reason: title is more important and we don't want mixed languages.

Reply:
> Hmm, maybe this is an irrelevant edge case, but if the default languages across regions differ, I guess we only want to have the default language of the region that is currently being sent? But ok, this would require some refactoring I guess. Probably we can just open a ticket for this once this PR is merged.

I had a go at it and it wasn't as convoluted as expected at all. Should I just include it or wait and make it a new PR as soon as this one is merged?

> 2 files changed, 68 insertions(+), 52 deletions(-)

Reply:
Lets fix this in a later/upcoming PR. I guess we need to move this to testing ASAP :see_no_evil: We should fix it soon, as we know this could be problematic. But right now all production regions are in Germany with the main language German.